sql-map-jquery-test/src/main/java/com/hazelcast/jet/tests/jquery/tests/JsonSorter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.jquery.tests;
+
+import java.util.Arrays;
+
+public final class JsonSorter {
+
+    private static JsonSorter instance;
+
+    private JsonSorter() {
+    }
+
+    public static JsonSorter getInstance() {
+        if (instance == null) {
+            instance = new JsonSorter();
+        }
+        return instance;
+    }
+
+    public String sortJsonAsCharArray(String jsonInputString) {
+        char[] chars = jsonInputString.toCharArray();
+        Arrays.sort(chars);
+        return new String(chars);
```

Comment:
This is really unusual, but I think it works :)

Reply:
yes, tested